engine/include/cubos/engine/render/camera/orthographic_camera.hpp

```diff
@@ -0,0 +1,41 @@
+/// @file
+/// @brief Component @ref cubos::engine::OrthographicCamera
+/// @ingroup render-camera-plugin
+
+#pragma once
+
+#include <cubos/core/reflection/reflect.hpp>
+
+#include <cubos/engine/api.hpp>
+
+namespace cubos::engine
+{
+    /// @brief Component which defines parameters of a orthographic camera used to render the world.
+    /// @note Should be used with @ref LocalToWorld.
+    /// @ingroup render-camera-plugin
+    struct CUBOS_ENGINE_API OrthographicCamera
+    {
+        CUBOS_REFLECT;
+
+        /// @brief The axis that can be fixed for this projection.
+        enum class Axis
+        {
+            HORIZONTAL,
+            VERTICAL

```

Comment:
```suggestion
            Horizontal,
            Vertical
```
I dunno why clang-tidy didn't catch this, but we use PascalCase for enum variants.